### PR TITLE
Removes "generic event spawn" landmarks from maps and code

### PR DESCRIPTION
**Pull request by coiax**
They were entirely unused apart from three uses by Clockwork Cult to
place things randomly on the station. `find_safe_turf()` does this
instead, without requiring so much mapping effort.

Why? Because they weren't doing anything meaningful anyway, and the less
work for mappers, the better.

### CODE REVIEW

**Comment by 81Denton** [comment]
What about blobstart landmarks? They should be fine to remove as well (blobs pick their spawn turf now), unless they're used somewhere else.

**Review by ShizCalev — CHANGES_REQUESTED** [review]
Gotta rework this too since you're removing a **VAST** majority of their spawn points. (blobspawn isn't even in this list to start with.)

https://github.com/tgstation/tgstation/blob/b5c2eddf217c91731c6baeaa4a03704eaa6f0188/code/modules/holiday/easter.dm#L24-L31

File: code/modules/antagonists/clockcult/clock_structures/ark_of_the_clockwork_justicar.dm [hunk 1]
```diff
@@ -301,8 +302,8 @@
 	switch(progress_in_seconds)
 		if(-INFINITY to GATEWAY_REEBE_FOUND)
 			if(!second_sound_played)
-				for(var/V in GLOB.generic_event_spawns)
-					addtimer(CALLBACK(src, .proc/open_portal, get_turf(V)), rand(100, 600))
+				for(var/i in 1 to PORTAL_COUNT)
```
Comment (ShizCalev) [inline code/modules/antagonists/clockcult/clock_structures/ark_of_the_clockwork_justicar.dm]: this is a major rebalance to clockcult. went from having a few hundred portals around the station to only 20.

File: code/modules/antagonists/clockcult/clock_structures/ark_of_the_clockwork_justicar.dm [hunk 2]
```diff
@@ -162,7 +163,7 @@
 			var/atom/movable/target = L
 			if(isobj(L.loc))
 				target = L.loc
-			target.forceMove(get_turf(pick(GLOB.generic_event_spawns)))
+			target.forceMove(find_safe_turf())
```
Comment (ShizCalev) [inline code/modules/antagonists/clockcult/clock_structures/ark_of_the_clockwork_justicar.dm]: Note this does have the potential to now randomly teleport players into say, the SME's rad collector areas, or the gravity generator chamber, which would more than likely kill them with radiation. Could also just straight up dump them into the middle of the AI core, AI upload, ect.

File: code/modules/antagonists/clockcult/clock_structures/ark_of_the_clockwork_justicar.dm (hunk 1 above)
Comment (coiax) [reply code/modules/antagonists/clockcult/clock_structures/ark_of_the_clockwork_justicar.dm]: Not an intentional rebalance. Should I determine the number of portals based on the number of floor turfs on the station z level or something?

File: code/modules/antagonists/clockcult/clock_structures/ark_of_the_clockwork_justicar.dm (hunk 2 above)
Comment (coiax) [reply code/modules/antagonists/clockcult/clock_structures/ark_of_the_clockwork_justicar.dm]: Pretty sure that people are only chucked out of Reebe when the ark finishes or blows up, and then the round's over.

**Comment by coiax** [comment]
I was unaware that generic event spawn landmarks were actually in use.

**Pull request closed without merging**